code/game/objects/items/theft_items.dm

```diff
@@ -253,7 +253,7 @@
 
 /obj/item/nuke_core_container/supermatter
 	name = "supermatter bin"
-	desc = "A tiny receptacle that releases an inert hyper-noblium mix upon sealing, allowing a sliver of a supermatter crystal to be safely stored."
+	desc = "A modified Cybersun Industries secure hazardous waste container, with an experimental alloy interior designed to hold Supermatter crystals."
```

Comment:
nobelium gas is used for materials that can touch the supermatter. See tongs / scalpel.